clio-util/src/main/scala/org/broadinstitute/clio/util/model/DataType.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.clio.util.model
+
+import enumeratum.{Enum, EnumEntry}
+import scala.collection.immutable.IndexedSeq
+
+sealed trait DataType extends EnumEntry
+
+object DataType extends Enum[DataType] {
+  override val values: IndexedSeq[DataType] = findValues
+
+  case object WGS extends DataType
+  case object Exome extends DataType
+  case object RNA extends DataType
+  case object Genome extends DataType
```

Comment:
From slack: it shouldn't cause server failures to limit these to just WGS and Exome, so let's do it and prevent any more RNA/Genome entries from getting loaded in.

Reply:
Before we merge this we should upsert some RNA records to dev, so we can test querying them post-merge.